.ci/docker/common/install_cuda.sh

```diff
@@ -10,6 +10,7 @@ else
   arch_path='sbsa'
 fi
 
+CUDNN_VERSION=9.14.0.64
```

Comment:
let's revisit this as I was reminded sm70 was readded to CUDA 12 in PyTorch but removed in cuDNN starting with 9.11
https://github.com/pytorch/pytorch/issues/162574